options.js

```diff
@@ -66,3 +66,20 @@ exports.getDefaults = function getDefaults(features) {
 
   return options;
 };
+
+exports.getMinifier = function getMinifier() {
```

Comment:
You actually are totally right, I started to do that and then I was like no that wouldn't be any good.